lib/Cake/Console/Command/TestShell.php

```diff
@@ -226,6 +226,8 @@ protected function _runnerOptions() {
 
 		if (!empty($params['no-colors'])) {
 			unset($params['no-colors'], $params['colors']);
+		} elseif (strtolower(substr(php_uname(), 0, 3)) === 'win') {
```

Comment:
There is an existing mention of it related to [disabling colors on windows](http://book.cakephp.org/3.0/en/console-and-shells.html#turning-off-colouring).